docs/source/terms_and_model.rst

```diff
@@ -435,14 +432,15 @@ objects (which would otherwise be represented using symbolic shorthand).
   set of alleles at multiple locations and/or with ploidy other than
   two. VRS Genotype entity is based on this broader definition.
 * The term "diplotype" is often used to refer to two in-trans haplotypes at a locus.
-  VRS Genotype entity subsumes the conventional definition of diplotype. Therefore,
+  VRS Genotype entity subsumes the conventional definition of diplotype, though
+  it describes no explicit in-trans phase relationship. Therefore,
   VRS does not include an explicit entity for diplotypes. See :ref:`this note
   <genotypes-represent-haplotypes-with-arbitrary-ploidy>` for a discussion.
 * VRS makes no assumptions about ploidy of an organism or individual nor any
   polysomy affecting a locus. The `genotype.count` attribute explicitly captures the total
-  count of in-trans molecules at a genomic locus represented by the Genotype.
+  count of molecules associated with a genomic locus represented by the Genotype.
 * In diploid organisms, there are typically two instances of each autosomal chromosome,
-  and therefore two instances of sequence at a particular location. Thus, Genotypes will
+  and therefore two instances of sequence at a particular locus. Thus, Genotypes will
```

Comment:
I'd add that if the desire is to express a specific diplotype, it could be represented as a genotype of two haplotypes.

Reply:
Sure! Added some text for this in 40ef2af.